split.Console/Models/Blackjack/BlackjackGame.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+
+namespace split.Console.Models.Blackjack
+{
+    public class BlackjackGame : Game
+    {
+        Deck CardDeck;
+        BlackjackPlayer Dealer;
+
+        public BlackjackGame(int decks) : base(GameType.Blackjack)
+        {
+            CardDeck = new Deck(decks);
+            Dealer = new BlackjackPlayer("Dealer", null);
+        }
+
+        /// <summary>
+        /// Adds a newly created player to the game
+        /// </summary>
+        /// <param name="newPlayer">Created player to be added to the game</param>
+        public void AddPlayer(BlackjackPlayer newPlayer)
+        {
+            Players.Add(newPlayer);
+        }
+
+        /// <summary>
+        /// Puts the game into an active state
+        /// </summary>
+        /// <returns>True is game has been started</returns>
+        public new bool StartGame()
+        {
+            CardDeck.Shuffle();
+            return base.StartGame();
+        }
+
+        /// <summary>
+        /// Deals 2 Cards to each player including the dealer
+        /// </summary>
+        public void DealRoundOfCards()
```

Comment:
As we talked before, for this method to be testable, it's best it returns a single Player object with cards.. or maybe even just a single card. You can move the loops into Manager.